docs/Open Data Model/Open Data Model.md

```diff
@@ -0,0 +1,755 @@
+
+ - File context data** (ROADMAP ITEM)**
+This data includes contextual information about files and can be sourced from systems such as FireEye, Application Control and others.
+
+ - Threat intelligence context data **(ROADMAP ITEM)**
+This data includes contextual information about URLs, domains, websites, files and others.
+
+**Naming Convention**
+-----------------
+
+A naming convention is needed for the Open Data Model to represent common attributes across vendor products and technologies. The naming convention is described below.
+
+**Prefixes**
+--------
+
+|  Prefix | Description  |  
+|---|---|
+|  src | Corresponds to the “source” fields within a given event (i.e. source address)|  
+|  dst | Corresponds to the “destination” fields within a given event (i.e. destination address) |  
+|  dvc | Corresponds to the “device” applicable fields within a given event (i.e. device address) and represent where the event originated  |  
+| fwd  | Forwarded from device   |  
+| request | Corresponds to requested values (vs. those returned, i.e. “requested URI”) |  
+| response  | Corresponds to response value (vs. those requested) |  
+| file  |  Corresponds to the “file” fields within a given event (i.e. file type) |  
+| user  | Corresponds to user attributes (i.e. name, id, etc.)  |  
+| xlate  | Corresponds to translated values within a given event (i.e. src_xlate_ip for “translated source ip address” |  
+| in  | Ingress|  
+| out | Egress |  
+| new | New value |  
+| orig | Original value |  
+| app | Corresponds to values associated with application events |  
+
+
+**Security Event Log/Alert Data Model**
+-----------------------------------
+
+The data model for security event logs/alerts is detailed in the below. The attributes are categorized as follows:
+
+ - Common -attributes that are common across many device types
+ - Device -attributes that are applicable to the device that generated the event
+ - File -attributes that are applicable to file objects referenced in the event
+ - Endpoint -attributes that are applicable to the endpoints referenced in the event
+ - User- attributes that are applicable to the user referenced in the event
+ - Proxy - attributes that are applicable to proxy events
+ - Protocol
+
+> DNS - attributes that are specific to DNS events
+> HTTP - attributes that are specific to HTTP events
+> SMTP, SSH, TLS, DHCP, IRC, SNMP and FTP
+
+Note: The model will evolve to include reserved attributes for additional device types that are not currently represented. The model can currently be extended to support ANY attribute for ANY device type by following the guidance outlined in the section titled **“Extensibility of Data Model”.**
+
+Note: Attributes denoted in BLUE represent those that are listed in the model multiple times for the purpose of
```

Comment:
I don't know if markdown can do color. If not, we should change BLUE to bold. And, change all the blue items in the original doc to be bolded.